src/Test/Unit/Util/CpuTest.php

```diff
@@ -0,0 +1,54 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+namespace Magento\MagentoCloud\Test\Unit\Util;
+
+use Magento\MagentoCloud\Shell\ShellInterface;
+use Magento\MagentoCloud\Util\Cpu;
+use PHPUnit\Framework\MockObject\MockObject;
+use PHPUnit\Framework\TestCase;
+use Psr\Log\LoggerInterface;
+
+/**
+ * @inheritdoc
+ */
+class CpuTest extends TestCase
+{
+    /**
+     * @var ShellInterface|MockObject
+     */
+    private $shellMock;
+
+    /**
+     * @var LoggerInterface|MockObject
+     */
+    private $loggerMock;
+
+    /**
+     * @var Cpu
+     */
+    private $cpu;
```

Comment:
Target class better to keep in top of properties, those won't move with adding/deleting them later